spec/models/category_spec.rb

```diff
@@ -1,5 +1,21 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe Category, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  
+  describe "Associations" do
+    it "has belong to parent" do
+      association = described_class.reflect_on_association(:parent)
+      expect(association.macro).to eq :belongs_to
```

Comment:
code dài quá
em dùng cú pháp `it {should belong_to :parent}` hoặc `it {is_expected.to belongs_to :parent}`

Reply:
ngoài ra thêm rspec cho validate. scope ... nữa nhé chứ ko chỉ association